marco/AutamationPractice/src/test/java/at05ui/hooks/hooks.java

```diff
@@ -0,0 +1,37 @@
+package at05ui.hooks;
+
+import at05ui.sampleapp.ui.IdentifyPage;
+import at05ui.sampleapp.ui.MyAccountPage;
+import at05ui.sampleapp.ui.MyAddressPage;
+import cucumber.api.java.After;
+
+public class hooks {
+
+  private MyAddressPage myAddressPage;
+  private MyAccountPage myAccountPage;
+  private IdentifyPage identifyPage;
+
+  public hooks(MyAddressPage myAddressPage, MyAccountPage myAccountPage, IdentifyPage identifyPage) {
+    this.myAddressPage = myAddressPage;
+    this.myAccountPage = myAccountPage;
+    this.identifyPage=identifyPage;
+  }
+
```

Comment:
Add comments for the methods in this classe

Reply:
I was working in this class, I forgot of put the comments , thank for the observation.
I already fixed.
Thank a lot.